internal/identify/identify.go

```diff
@@ -130,17 +179,20 @@ func (t *SceneIdentifier) getSceneUpdater(ctx context.Context, s *models.Scene,
 		ret.Partial.StudioID = models.NewOptionalInt(*studioID)
 	}
 
-	ignoreMale := false
-	for _, o := range options {
-		if o.IncludeMalePerformers != nil {
-			ignoreMale = !*o.IncludeMalePerformers
-			break
-		}
+	includeMalePerformers := true
+	if options.IncludeMalePerformers != nil {
+		includeMalePerformers = *options.IncludeMalePerformers
 	}
 
-	performerIDs, err := rel.performers(ctx, ignoreMale)
+	addSkipSingleNamePerformerTag := false
+	performerIDs, err := rel.performers(ctx, !includeMalePerformers)
 	if err != nil {
-		return nil, err
+		var singleErr *ErrSkipSingleNamePerformer
+		if errors.As(err, &singleErr) {
```

Comment:
With the above change, this can be changed to just be `errors.Is(err, ErrSkipSingleNamePerformer)`.